decidim/decidim
# Fix admin page titles

#### :tophat: What? Why?
This PR tries to ensure that all the admin pages do have a relevant page title.

- Relates to #11272

#### Testing
1. Login as admin 
2. Visit admin panel 
3. Browse trough pages and check the page title

:hearts: Thank you!
